src/NServiceBus.Azure.QuickTests/Subscriptions/When_subscribing.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace NServiceBus.AzureStoragePersistence.Tests
+{
+    using System;
+    using System.Linq;
+    using Unicast.Subscriptions;
+    using NUnit.Framework;
+
+    [TestFixture]
+    [Category("AzureStoragePersistence")]
+    public class When_subscribing : BaseAzureSagaPersisterTest
+    {
+        const string Q = "address://test-queue";
```

Comment:
Is VS intellisense on a strike and that's why we use one letter names?
While it's a minor, I still rather read a proper name. 😉